<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandWidget</class>
 <widget class="QWidget" name="CommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEditName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Match</string>
         </property>
         <property name="buddy">
          <cstring>lineEditMatch</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Icon:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditIcon</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEditIcon">
           <property name="toolTip">
            <string>Command name (shown in menu)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelIcon">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonBrowse">
           <property name="text">
            <string>&amp;Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Copy to &amp;tab</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxCopyToTab</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxCopyToTab">
         <property name="toolTip">
          <string>Name of tab to copy new items into (leave empty not to copy)</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditMatch">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use command only for items whose text match this regular expression.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Examples:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;  Match URL    &lt;span style=&quot; font-weight:600;&quot;&gt;^(https?|ftp)://&lt;/span&gt;&lt;/p&gt;&lt;p&gt;  Match PDF filenames    &lt;span style=&quot; font-weight:600;&quot;&gt;\.pdf$&lt;/span&gt;&lt;/p&gt;&lt;p&gt;  Match single character    &lt;span style=&quot; font-weight:600;&quot;&gt;^.$&lt;/span&gt;&lt;/p&gt;&lt;p&gt;  Match remote multimedia    &lt;span style=&quot; font-weight:600;&quot;&gt;^http://.*\.(ogv|vlc|mp4|mp3)$&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxIgnore">
           <property name="toolTip">
            <string>Don't save the matched clipboard content to history.</string>
           </property>
           <property name="text">
            <string>I&amp;gnore</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEditName">
           <property name="toolTip">
            <string>Command name (shown in menu)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxEnable">
           <property name="toolTip">
            <string>Enable this action</string>
           </property>
           <property name="text">
            <string>&amp;Enable</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Comma&amp;nd</string>
         </property>
         <property name="buddy">
          <cstring>lineEditCommand</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditCommand">
         <property name="toolTip">
          <string>Command to run. Use %1 for item text passed as argument.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCommandOptions">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Command options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBoxInput">
            <property name="toolTip">
             <string>Send item text to standard input of the command.</string>
            </property>
            <property name="text">
             <string>In&amp;put</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxOutput">
            <property name="toolTip">
             <string>Save standard output to items.</string>
            </property>
            <property name="text">
             <string>Ou&amp;tput into item(s)</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,4">
          <item>
           <widget class="QLabel" name="separatorLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Separator:</string>
            </property>
            <property name="buddy">
             <cstring>lineEditSeparator</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditSeparator">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Separator to match for splitting the output to multiple items.</string>
            </property>
            <property name="text">
             <string>\n</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelOutputTab">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Output &amp;tab:</string>
            </property>
            <property name="buddy">
             <cstring>comboBoxOutputTab</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxOutputTab">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Save items in tab with given name (leave empty to save in first tab)</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="checkBoxWait">
            <property name="toolTip">
             <string>Before executing the command show action dialog.</string>
            </property>
            <property name="text">
             <string>&amp;Wait</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxAutomatic">
            <property name="toolTip">
             <string>Run the command automatically</string>
            </property>
            <property name="text">
             <string>Au&amp;tomatic</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Shortcut:</string>
            </property>
            <property name="buddy">
             <cstring>pushButtonShortcut</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonShortcut">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>checkBoxEnable</tabstop>
  <tabstop>lineEditMatch</tabstop>
  <tabstop>checkBoxIgnore</tabstop>
  <tabstop>lineEditIcon</tabstop>
  <tabstop>pushButtonBrowse</tabstop>
  <tabstop>comboBoxCopyToTab</tabstop>
  <tabstop>lineEditCommand</tabstop>
  <tabstop>checkBoxInput</tabstop>
  <tabstop>checkBoxOutput</tabstop>
  <tabstop>lineEditSeparator</tabstop>
  <tabstop>comboBoxOutputTab</tabstop>
  <tabstop>checkBoxWait</tabstop>
  <tabstop>checkBoxAutomatic</tabstop>
  <tabstop>pushButtonShortcut</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxOutput</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSeparator</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
